tests/index.spec.js

```diff
@@ -408,6 +408,28 @@ describe('Testing generic actions', () => {
     expect(result.fee_collected).to.be.a('number')
   })
 
+  it(`getFee for addBundledTransactions`, async () => {
+    const result = await fioSdk.genericAction('getFeeForAddBundledTransactions', {
+      fioAddress: newFioAddress
+    })
+
+    expect(result).to.have.all.keys('fee')
+    expect(result.fee).to.be.a('number')
+  })
+
+  it(`add Bundled Transactions`, async () => {
+    const result = await fioSdk.genericAction('addBundledTransactions', {
+      fioAddress: newFioAddress,
+      bundleSets: bundleSets,
+      newOwnerKey: pubKeyForTransfer,
```

Comment:
the `bundleSets` is undefined, we need to set some value above, for example `const bundleSets = 3`
also there is no `newOwnerKey` prop for that method, I think we should remove this line